arrays/arrays.c

```diff
@@ -72,8 +87,17 @@ void resize_array(Array *arr) {
 char *arr_read(Array *arr, int index) {
 
   // Throw an error if the index is greater than the current count
+  if (index > arr->count) {
+    switch(errno) {
+      default:
+        printf("Reading index of array -> Error: Index is out of bounds \n");
+    }
+
+    return arr->elements[index-1];
```

Comment:
Interesting, not sure if I understand the function of this line what are you trying to return here?

Reply:
I removed it a while back. I realized I could just do exit(1) instead of returning so that is what I did in the updated code, which should already be commited. I was returning earlier because of compiler warnings.